cmake/platform_linux.cmake

```diff
@@ -53,9 +53,13 @@ function(setup_litecore_build_linux)
     )
 endfunction()
 
-function(setup_support_build)
+function(setup_support_build_linux)
```

Comment:
If you rename this then you need to add `setup_support_build` to `platform_linux_desktop` which just calls this method.